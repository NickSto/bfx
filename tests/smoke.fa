>read1 desc1
GATTACAGATTACA
>read2 desc2
TGTAATCTGTAATC
>read3 desc3
TAGACATTATGTTT
