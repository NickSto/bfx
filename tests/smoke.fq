@read1 desc1
GATTACAGATTACA
+
???BBBDDDBDDDD
@read2 desc2
TGTAATCTGTAATC
+
$^%Huh)ushv76J
@read3 desc3
TAGACATTATGTTT
+
)($J)F(JS(FJSE
