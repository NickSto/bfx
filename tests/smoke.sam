read1 desc1	99	ref	101	30	MIDSHIP	=	0	10	GATTACAGATTACA	???BBBDDDBDDDD
read2 desc2	99	ref	101	30	MIDSHIP	=	0	10	TGTAATCTGTAATC	$^%Huh)ushv76J	NM:i:1
read3 desc3	99	ref	101	30	MIDSHIP	=	0	10	TAGACATTATGTTT	)($J)F(JS(FJSE
