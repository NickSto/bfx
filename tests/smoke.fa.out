Reading input as format 'fasta'.
Read 1 id/name: 'read1'/'read1 desc1'
Read 1 seq:  'GATTACAGATTACA'
Read 1 qual: ''
Read 2 id/name: 'read2'/'read2 desc2'
Read 2 seq:  'TGTAATCTGTAATC'
Read 2 qual: ''
Read 3 id/name: 'read3'/'read3 desc3'
Read 3 seq:  'TAGACATTATGTTT'
Read 3 qual: ''
